index.html

```diff
@@ -37,6 +39,7 @@ <h2 class="title-bar-item">JSON Path Finder</h2>
         <button class="editor-button-row-button" @click="loadSample">Sample</button>
         <button class="editor-button-row-button" @click="beautify">Beautify</button>
         <button class="editor-button-row-button" @click="minify">Minify</button>
+        <button class="editor-button-row-button" @click="darkmode">Dark Mode</button>
```

Comment:
Two requests:

1. Could you place the button either in the header to the left of the github logo/link, or on the far right side of the footer? Flat button with the same background color as the header/footer.
2. Can you have the text toggle between `Light Mode` and `Dark Mode` depending on which mode is active? I _think_ that should be doable with Vue's templating, but the last time I used Vue was when I made this 6 years ago... No stress if it's a big lift.